src/dart-2.md

```diff
@@ -27,11 +29,18 @@ For information on potential changes, see the
   * [Obsolete features](#obsolete-features)
   * [Strong mode and static typing](#strong-mode-and-static-typing)
 
-<aside class="alert alert-info" markdown="1">
-**Flutter note:** You don't need to do anything to prepare Flutter code
-for Dart 2. If changes become necessary,
-we'll add Flutter details to this page.
-</aside>
+
+## Migration tips: Flutter
+
+Use `--preview-dart-2` with the `flutter` tool to enable the latest
```

Comment:
Can we confirmed that the flag name will be `--preview-dart-2` across all platforms?
(It currently seems to be `--preview_dart_2` for Dart VM.)

Reply:
From all the emails I've seen, the flag name is to be standardized as `--preview-dart-2`.

/cc @kevmoo